examples/Contact/create-contact.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @package Redbitcz\Vyfakturuj\SimpleShopAPI
```

Comment:
Všude je v package slovo Vyfakturuj. To by být asi nemuselo.

Reply:
Zkusil jsem `\Redbitcz\SimpleShop\Api`, ale když rozhodnete jinak - změním ho

Reply:
Za mě Ok.

Reply:
done